First update for 2020

william-liebenberg opened the pull request:
updated dates
updated venue
link to new prospectus
update logo to 2020
comment out old agenda
comment out old speakers
comment out old sponsors
update cfp section
update top-nav to hide speakers, show cfp
minor style tweak for mobile

new look:
![serverless-homepage-2020](https://user-images.githubusercontent.com/23464439/74604939-1fedd780-5117-11ea-9c87-2e6021106db6.jpg)


current look:
![serverless-homepage-2019](https://user-images.githubusercontent.com/23464439/74604941-2419f500-5117-11ea-9905-f8595b626270.jpg)

william-liebenberg committed:
update all dates
update venue
link to new prospectus
update logo to 2020

william-liebenberg committed:
include prospectus
include 2020 logo

william-liebenberg committed:
tweak date card - display on right
tweak melbourne image - change blending to show mascots nicely
comment out old agenda
comment out old speakers
comment out old sponsors
update cfp section
update top-nav to hide speakers, show cfp

arthurscheeren commented on src/site/config.toml:
```diff
@@ -6,38 +6,38 @@ title = "ServerlessDays Melbourne"
     logo = "/images/serverless_logo_melb.png"
     description = "A community based conference focusing on Serverless based platforms and technologies. Melbourne, August 29, 2019"
```
We should also change it in here the 2019 description to "August 20th, 2020" ?

william-liebenberg replied:
Fixed :)

william-liebenberg committed:
fix conf date

mattgillard reviewed (commented):
* remove ticket price
* Early bird tickets on sale (March 14) - maybe we can soft launch a week earlier

Other than that - looks good!

ajhooker82 reviewed (approved):
Looks good Will/Arthur. Good stuff. Just needs the updated links for ticketing, but thats to come of course.

william-liebenberg committed:
fix logo color for MELBOURNE
add extra backgrounds
tweak image styling
use more mascots
add draft tickets links

william-liebenberg committed:
remove ticket price (for now)

william-liebenberg committed:
format dates consistently (month, day, year)
add Early Bird ticket date

william-liebenberg committed:
resize background image

william-liebenberg committed:
hide diversity tickets link for now

william-liebenberg committed:
make background image a tad smaller - for faster loading

william-liebenberg committed:
update ticket link

william-liebenberg committed:
change ticket sales date
